### PR TITLE
News admin page danya

### PR DESCRIPTION
**Pull request by DanyilTerentiev**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)
#289 

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/router/Routes.tsx
Comment (ormykhalyshyn) [inline src/app/router/Routes.tsx:1]: Return Protected Component

File: src/features/AdditionalPages/NewsPage/News.component.tsx
Comment (ormykhalyshyn) [inline src/features/AdditionalPages/NewsPage/News.component.tsx:1]: consol.logs, comments

File: src/features/AdditionalPages/NewsPage/Title/Title.component.tsx
Comment (ormykhalyshyn) [inline src/features/AdditionalPages/NewsPage/Title/Title.component.tsx:1]: ...

**Review by ormykhalyshyn — CHANGES_REQUESTED** [review]

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@use "@assets/sass/variables/_variables.fonts.scss" as ft;
+@use "@assets/sass/_utils.functions.scss" as f;
+@use "@assets/sass/mixins/_utils.mixins.scss" as mut;
+
+.partnersContainer{
+    .partnersBlock{
+        .keyPartnersBlock{
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:7]: spaces

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            display: flex;
+            column-gap: f.pxToRem(30px);
+            row-gap: f.pxToRem(80px);
+            flex-wrap: wrap;
+            justify-content: center;
+
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:13]: maybe some mixins here?

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            margin-bottom: f.pxToRem(80px);
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:14]: and here

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+            .partnersItem{
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:16]: space

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                @include mut.sized(380px, 110px);
+            }
+        }
+        
+        .otherPartnersBlock{
+            display: flex;
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:22]: here

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            column-gap: f.pxToRem(10px);
+            row-gap: f.pxToRem(80px);
+            flex-wrap: wrap;
+            justify-content: center;
+
+            margin-bottom: f.pxToRem(60px);
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:28]: here

File: src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            .partnersItem{
+                @include mut.sized(232px, 68px);
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 1020px) {
+    .partnersContainer .partnersBlock{
+        .keyPartnersBlock{
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersBlock/PartnersBlock.styles.scss:39]: and here

File: src/features/AdditionalPages/NewsPage/PartnersItem/PartnersItem.styles.scss [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+@use "@assets/sass/_utils.functions.scss" as f;
+@use "@assets/sass/mixins/_utils.mixins.scss" as mut;
+
+.partnersItem{
+    text-align: center;
+    position: relative;
+
+    img{
+        height: 100%;
+        width: 100%;
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/PartnersItem/PartnersItem.styles.scss:11]: mixin

File: src/features/AdditionalPages/NewsPage/Title/Title.styles.scss [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+@use "@assets/sass/mixins/_utils.mixins.scss" as mut;
+
+.partnersContainer{
+    .titleContainer{
+        margin-bottom: f.pxToRem(24px);
+    
+        .titleSmall{
+            @include mut.with-font($font-family: ft.$roboto-font, $font-weight: 300, $font-size: 12px);
+            line-height: 14px;
+            color: #891F16;
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/Title/Title.styles.scss:12]: crime

File: src/features/AdditionalPages/NewsPage/Title/Title.styles.scss [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+            } 
+        }       
+    }
+}
+
+@media screen and (max-width: 680px) {
+    .partnersContainer{
+        .titleContainer{
+            .content .logo{
+                top:  f.pxToRem(-50px);
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/Title/Title.styles.scss:65]: and here

File: src/features/AdditionalPages/NewsPage/Title/Title.styles.scss [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+                right:  f.pxToRem(-80px);
```
Comment (Katerix) [inline src/features/AdditionalPages/NewsPage/Title/Title.styles.scss:66]: here

File: src/features/AdminPage/NewsPage/Partners.styles.scss [hunk 12]
```diff
@@ -0,0 +1,42 @@
+@use "@sass/mixins/_utils.mixins.scss" as mut;
+@use "@sass/variables/_variables.fonts.scss" as ft;
+@use "@sass/variables/_variables.colors.scss" as c;
+@use "@sass/_utils.functions.scss" as f;
+
+.partners-page{
+    padding-top: f.pxToRem(100px);
+    display: flex;
+    gap: f.pxToRem(10px);
+    .partners-page-add-button{
```
Comment (Katerix) [inline src/features/AdminPage/NewsPage/Partners.styles.scss:10]: pls extra line after line 9

**Review by Katerix — CHANGES_REQUESTED** [review]
Good job! Please read my comments below and be attentive to code styling! You are a smartass ))

File: src/app/common/constants/api-routes.constants.ts [hunk 13]
```diff
@@ -12,6 +12,14 @@ export const API_ROUTES = {
         UPDATE: 'fact/update',
         DELETE: 'fact/delete',
     },
+    NEWS: {
+        GET_ALL: 'news/getAll',
+        GET: 'news/getById',
```
[the remark this answers is not in the document]
Comment (Lolimkeri) [reply src/app/common/constants/api-routes.constants.ts:17]: I checked other api routes and they used GET to call getById

**Review by DanyilTerentiev — COMMENTED** [review]
Please check my pr. I cleaned another's files, so there are only mine. Also cleaned all logs which saw. Waiting for your review!

**Review by ormykhalyshyn — APPROVED** [review]

**Pull request merged**